suite-native/module-device-onboarding/src/screens/FirmwareInstallationScreen.tsx

```diff
@@ -8,6 +9,7 @@ export const FirmwareInstallationScreen = () => {
     const { handleExitButtonPress } = useExitAlert();
     const { navigateToNextScreenAfterFirmwareInstallation } =
         useNavigateToNextScreenAfterFirmwareInstallation();
+    useOverrideBackNavigation();
```

Comment:
I would first merge this [PR](https://github.com/trezor/trezor-suite/pull/21983) where we can use the new useInterceptNativeNavigation hook to handle iOS gestures everywhere. WDYT @juriczech

Reply:
Yeh its hacky and I meant to remove useOverrideBackNavigation(); and block only native navigations via the proposed hook in the new PR, because `useOverrideBackNavigation` intercepts ALL navigation and it’s only by luck that `handleExitButtonPress` works, since it uses `pop` instead of `goBack`. 😄

Reply:
It's not by luck. @yanascz Actually spent a lot of time tweaking the behavior 😄 But yeah, we will need to figure out where it makes sense to use the hook and where it doesn't ;)